packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -206,6 +206,24 @@ A 'render prop' function that can be used to customize the block's appender.
 />
 ```
 
+### `__experimentalCaptureDescendantsToolbars`
```

Comment:
Do we need both use cases __experimentalCaptureDescendantsToolbars, and __experimentalCaptureChildToolbar? Any block you have in mind where we will use __experimentalCaptureChildToolbar but not __experimentalCaptureDescendantsToolbars?
If using both options would it make sense to use a single prop __experimentalCaptureToolbar, with possible values being 'ancestor', 'child'?

Reply:
At the current time, I can't actually think of any use case for the immediate child toolbar capture. Perhaps we should just standardise on `__experimentalCaptureDescendantsToolbars`?

Reply:
I've decided to normalise to a single API to capture all toolbars. The immediate child API is now removed.